### ISSUE TITLE
[Question] How to always show all labels of Axis?

### ISSUE THREAD

**Issue by unofficialdev** [beto-rodriguez/LiveCharts2#90]
I am using 1 Cartesian Chart (Line Series) in avalonia. my question is how to always show all labels of Axis, or show all labels when zoomed in.
I haven't been able to find any options for this.

**Comment by beto-rodriguez** [comment]
Please add an image that compares both, the result you have, and the result you spect

**Comment by unofficialdev** [comment]
Not show all label:
![Not Show All Labels](https://user-images.githubusercontent.com/43975116/125966697-8a7fa85e-4b14-430a-bb36-a94398979363.png)

Show all label:
![Show All Labels](https://user-images.githubusercontent.com/43975116/125966703-7fdc96e4-b2b7-48d1-b006-f6cec0970b2b.png)

i get all labels only when zoomed in all the way, my problem is want to show all labels even if not zoomed in all the way

**Comment by beto-rodriguez** [comment]
Set the `MinStep` property to `1` (https://github.com/beto-rodriguez/LiveCharts2/blob/master/docs/components/1.7.axes.md#minstep-property)

``` xml
<lvc:CartesianChart 
    Series="{Binding Series}"
    XAxes="{Binding XAxes}"
    YAxes="{Binding YAxes}">
</lvc:CartesianChart>
```

```c#
XAxes = new List<Axis>
            {
                new Axis
                {
                    // The MinStep property lets you define the minimum separation (in chart values scale)
                    // between every axis separator, in this case we don't want decimals,
                    // so lets force it to be greater or equals than 1
                    MinStep = 1
                }
            };
```

full sample:
https://github.com/beto-rodriguez/LiveCharts2/blob/a53cb80a336f0c068727875ebf0bbfb049506075/samples/ViewModelsSamples/Axes/LabelsRotation/ViewModel.cs

https://github.com/beto-rodriguez/LiveCharts2/blob/master/samples/AvaloniaSample/Axes/LabelsRotation/View.axaml

**Comment by unofficialdev** [comment]
Thanks for your enthusiastic support, I would like to ask 1 more question.
at the bottom of the charts label always shows the number of the data set, the number 100 it's not the label. how to turn it off?
![Image 8](https://user-images.githubusercontent.com/43975116/125974521-3b516b40-322c-4cb9-be81-ea8207f9c6c6.png)

**Comment by beto-rodriguez** [comment]
since there is no label in your array at the position 100, then the axis is building a label based on the index of the element, I think it would be better to return an empty string in this case

**Comment by unofficialdev** [comment]
thank you very muc

**Comment by unofficialdev** [comment]
@beto-rodriguez 

after getting your answer, I checked the label list again. still full 100 labels corresponding to 100 values. but at the end still shows "100". I think the problem is not because I don't have enough 100 labels
![Image 111](https://user-images.githubusercontent.com/43975116/125985565-c4b8927a-e542-415d-b683-a31ed714ed35.png)

**Comment by beto-rodriguez** [comment]
I know, I meant that this issue is caused by design, this will be improved in a future release, a work around for now it to add multiple empty labels at the end but that is just to get rid of that, to fix this property we need to fix how the axis handles the labels

**Comment by beto-rodriguez** [comment]
This should be fixed with the refenced commit, this fix is included since `v2.0.beta.33`.

Thanks for the report.
